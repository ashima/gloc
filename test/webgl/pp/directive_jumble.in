#version 100
/*comment*/#error foo
#/*again*/error bar
#error baz/*third*/
