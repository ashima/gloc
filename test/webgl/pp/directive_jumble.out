File 0, line 2, col 11 - 21:
/*  comment */ #error foo

File 0, line 3, col 9 - 19:
/*  again */ #error bar

File 0, line 4, col 0 - 10:
#error baz/*third*/

Fatal: unrecoverable preprocessor error (4)
